### PR TITLE
About page in project site

### PR DESCRIPTION
**Pull request by mhausenblas**
This addresses #18

### CODE REVIEW

File: docs/content/about.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# About
+
+With polly, we're building on the shoulder of giants, or in other words: this
+is not the first attempt to make o11y tooling templating work. It all started out in
```
Comment (metalmatze) [inline docs/content/about.md:4]:
```suggestion
is not the first attempt to make observability tooling templating work. It all started out in
```
Let's spell it out to not confuse people.

File: docs/content/about.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+> A mixin is a set of Grafana dashboards and Prometheus rules and alerts, 
+> packaged together in a reuseable and extensible bundle. Mixins are written 
+> in jsonnet, and are typically installed and updated with jsonnet-bundler.
+
+The Mixins, based on jsonnet and maintained alongside the tool turned out to
+be a great idea, however, over time certain challenges and limitations became
+apparent.
+
+These shortcomings led to a desire to improve on the existing work on Mixins,
+resulting in what we now know as polly. Initially simply called Mixins-NG,
```
Comment (metalmatze) [inline docs/content/about.md:18]:
```suggestion
resulting in what we now know as polly. Initially simply called Mixins-NG (next generation), 
```

File: docs/content/about.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+> packaged together in a reuseable and extensible bundle. Mixins are written 
+> in jsonnet, and are typically installed and updated with jsonnet-bundler.
+
+The Mixins, based on jsonnet and maintained alongside the tool turned out to
+be a great idea, however, over time certain challenges and limitations became
+apparent.
+
+These shortcomings led to a desire to improve on the existing work on Mixins,
+resulting in what we now know as polly. Initially simply called Mixins-NG, 
+the work around polly kicked off publickly in [April 2021][polly-kickoff], with
```
Comment (metalmatze) [inline docs/content/about.md:19]:
```suggestion
the work around polly kicked off publicly in [April 2021][polly-kickoff], with
```

File: docs/content/about.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+At that point, Sam already had been working on the concept and core formalization
+for several months, broadening the group of folks involved, initially a small 
+group of interested folks, a mixture of past and future stakeholders and 
+contributors to Mixins.
+
+At this point, polly doesn't have a formal governance body or rule set, but
+we do have:
+
+* A guide on how to [contribute][contrib] to polly.
+* The commitment that polly and the artefacts in the organization are available
```
Comment (metalmatze) [inline docs/content/about.md:31]:
```suggestion
* The commitment that polly and the artifacts in the organization are available
```

Comment (mhausenblas) [reply docs/content/about.md:31]: Fair enough, I suppose we're aiming to have US rather than UK spelling then ;)

File: docs/content/about.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+The Mixins, based on jsonnet and maintained alongside the tool turned out to
+be a great idea, however, over time certain challenges and limitations became
+apparent.
+
+These shortcomings led to a desire to improve on the existing work on Mixins,
+resulting in what we now know as polly. Initially simply called Mixins-NG (next generation), 
+the work around polly kicked off publicly in [April 2021][polly-kickoff], with
+Sam Boyer's mail to the newly created Mixins mailing list. 
+
+At that point, Sam already had been working on the concept and core formalization
```
Comment (sdboyer) [inline docs/content/about.md:22]:
```suggestion
At that point, sam already had been working on the concept and core formalization
```

File: docs/content/about.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+we do have:
+
+* A guide on how to [contribute][contrib] to polly.
+* The commitment that polly and the artifacts in the organization are available
+  under Apache License Version 2.0 and will continue to be so.
+
+For any questions or suggestions, we ask you to use the [polly project
+discussion][discussion] section in the main repo.
+
+_Sam, Matthias, Michael_
```
Comment (sdboyer) [inline docs/content/about.md:37]:
```suggestion
_sam, Matthias, Michael_
```

**Review by sdboyer — CHANGES_REQUESTED** [review]
just my name - other than that, looks good! :)

**Pull request merged**